### DISCUSSION TITLE
Firmware Update bleibt hängen.

### DISCUSSION THREAD

**Discussion by Yosh1977** [tbnobody/OpenDTU#931] [Q&A]
Moin moin,

kann mir einer sagen warum das Firmware Update einfach hängen bleibt. 
mich hatte es schon mal, bei 35% und nichts passiert mehr. Oder bei 100%. 

Gibts dafür eine Erklärung ?

Gruß 
Yoshi

**Comment by Yosh1977** [comment]
Anderer Browser hat geholfen.

**Reply by pblanchardch** [reply]
Bisher hat der Update über den Edge Browser immer funktioniert. Nun, installiert habe ich [v23.6.9], funktioniert der Firmware update nicht mehr. Update wird gestartet und bleibt anschließend hängen. Getestet auf Edge, Firefox und Chrome.
